File: cs/2.methods/default.cs
namespace default
{
  class def
  {
    static void Home(string country = "United States")
    {
      Console.WriteLine("Anonymous User is from: " + country);
    }
    static void Main()
    {
      Home();
      Home("Poland");
      Home("Germany");
    }
  }
}

File: cpp/2.functions/default.cpp
#include <iostream>
using namespace std;

void Home(string name, string country = "United States") {
  cout << name << "'s home country is " << country << '.\n';
}

int main() {
  Home("Mevia");
  Home("Alexander", "Germany");
  Home("Felix", "Poland");
  return 0;
}
